File: apps/net/Ch4_OOP/Models/ElectricCar.cs
using System;
namespace Ch4_OOP.Models
{
    public class ElectricCar : Car
    {
        public ElectricCar(string modeName, float basePrice) :
            base(modeName, "electric", 0, basePrice)
        {
        }

        public override void Refuel()
        {
            Console.WriteLine($"ElectricCar: The {ModelName} is charging up.");
        }

        public override void Drive()
        {
            Console.WriteLine($"ElectricCar: The electric {ModelName} zooms silently along!");
        }
    }
}

File: apps/py/ch04_oop/models/electric_car.py
from models.car import Car


class ElectricCar(Car):

    def __init__(self, model_name: str, base_price: float):
        # first!
        super().__init__(model_name, 'electric', 0, base_price)
        # then other stuff..

    def drive(self):
        print(f"ElectricCar: The electric {self.model_name} zooms silently along!")

    def refuel(self):
        print(f'ElectricCar: The {self.model_name} is charging up.')
